### DISCUSSION TITLE
Alert showing on every page load OOM command not allowed when used memory > 'max memory'

### DISCUSSION THREAD

**Discussion by raldred** [outline/outline#1937] [Self Hosting] [answered]
Updated from 0.51.0 to 0.53.0 and everyone has starting seeing this notice showing up in the bottom left corner:
![image](https://user-images.githubusercontent.com/37096/110121971-f140ae80-7db6-11eb-89c0-bb74c6e83146.png)

Anyone know what this is?
There's plenty of free memory on the instance we're running outline one.

Thanks

**Comment by tommoor** [comment, marked as answer]
Your Redis instance is out of memory.

**Comment by raldred** [comment]
Ahh, I'd completely forgotten to check Redis.
Thank you.
